docs/reference-architectures/virtual-machines-linux/single-vm.md

```diff
@@ -15,25 +15,25 @@ pnp.series.prev: ./index
 
 # Run a Linux VM on Azure
 
-This reference architecture shows a set of proven practices for running a Linux virtual machine (VM) on Azure. It includes recommendations for provisioning the VM along with networking and storage components. This architecture can be used to run a single instance, and is the basis for more complex architectures such as n-tier applications. [**Deploy this solution**.](#deploy-the-solution)
+This reference architecture follows a set of proven practices for running a Linux virtual machine (VM) on Azure. It includes recommendations for provisioning the VM along with networking and storage components. This architecture can be used to run a single VM instance, and is the basis for more complex architectures such as N-tier applications. [**Deploy this solution.**](#deploy-the-solution)
```

Comment:
I'd like to keep this as "shows" because the primary goal of the doc is to teach/provide guidance.

Reply:
Agree

Reply:
Or how about "demonstrates"?

Reply:
either seems OK to me.